air-quality-backend/system_tests/in_situ_etl_suite/open_aq_etl_tests.py

```diff
@@ -262,6 +262,7 @@ def test__in_situ_etl__timeouts_retry_twice_then_stop(
         main()
 
     assert "Response for London contained no results" in caplog.text
+    assert "URL was: https://api.openaq.org//v2/measurements?limit=3000" in caplog.text
```

Comment:
I think this has an extra /

Reply:
This was actually wrong in the code as well, have tweaked it there too